packages/atoms/src/Select.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
```

Comment:
Please check again agains Figma, from your screenshot it looks there are lot of inconsistencies. For example icon should be visible alongside label in Select. Select label has probably wrong color. Arrow icon looks bigger than in Figma and also different color etc.

Reply:
Everything fixed. I put arrow icon as medium as it doesn't fit any other icon sizes. I think the 2px difference is not really noticeable.
[ad89ef9](https://github.com/trezor/trezor-suite/pull/5506/commits/ad89ef9ca39b59768143099cf46c5230b79c28fb)

Reply:
Apart from icon alongside label - still working on that

Reply:
We are doing pixel perfect desing so it matters :D In case you are doing any changes always ask Ondra. If he won't be happy about it then we will need implement some custom sizing to Icon